src/DynamoCore/Library/LibraryServices.cs

```diff
@@ -418,6 +418,37 @@ public FunctionDescriptor GetFunctionDescriptor(string managledName)
         }
 
         /// <summary>
+        ///     Returns a list of function descriptors associated with the function name.
+        /// </summary>
+        /// <param name="mangledName"></param>
+        /// <returns></returns>
+        public IEnumerable<FunctionDescriptor> GetAllFunctionDescriptors(string mangledName)
+        {
+            IEnumerable<FunctionDescriptor> descriptors = null;
+            string qualifiedName = mangledName.Split('@')[0];
+            FunctionGroup functionGroup;
+
+            // Check through both builtinFunctionGroups and importedFunctionGroups to find the function descriptors
+            if (builtinFunctionGroups.TryGetValue(qualifiedName, out functionGroup))
+            {
+                functionGroup = builtinFunctionGroups[qualifiedName];
```

Comment:
This statement is not required. If `TryGetValue` returns true, it will also return the value of the out parameter, which is `functionGroup` in this case.